### DISCUSSION TITLE
Supported Flink Version

### DISCUSSION THREAD

**Discussion by debakkerb** [spotify/flink-on-k8s-operator#291] [Q&A]
I'm currently using Flink v1.10.1, but I was wondering if it's possible to upgrade it.  I can't find anything in the codebase that is directly tied to a Flink version.  However, when I try to use Flink v1.13.5 for example, I get the following error message:

```
RuntimeError: Unable to fetch remote job server jar at https://repo.maven.apache.org/maven2/org/apache/beam/beam-runners-flink-1.13-job-server/2.30.0/beam-runners-flink-1.13-job-server-2.30.0.jar: HTTP Error 404: Not Found
```

That is correct, that JAR doesn't exist.  But I'm wondering where it tries to download that file and how the underlying mechanisms construct that URL.  I couldn't find anything in the codebase, so a steer would be more than welcome.

**Comment by regadas** [comment]
Hi @debakkerb! are you following some examples? Can you share your definition?
